docs/strategies/retry.md

```diff
@@ -156,15 +156,25 @@ Step 1: Calculating the base delay:
 
 - If `UseJitter` is set to `false` and `Delay` is specified then `Delay` multiplied by the actual attempt number will be used.
 - If `UseJitter` is set to `true` and `Delay` is specified then a random value is added to the `Delay` multiplied by the actual attempt number.
-  - The random value is between -25% of the newly calculated `Delay` and +25% of the newly calculated `Delay`.
+  - The random value is between -25% and +25% of the newly calculated `Delay`.
 
 > [!NOTE]
-> Because the jitter calculation is based on the newly calculated delay that's why the new delay could be smaller than the previous one.
+> Because the jitter calculation is based on the newly calculated delay, the new delay could be less than the previous value.
 
 Step 2 and 3 are the same as for the Constant algorithm.
 
 ### Exponential
 
+This algorithm increases the delays for every attempt in an exponential fashion if no jitter is used.
+
+- If `UseJitter` is set to `false` and `Delay` is specified then squaring actual attempt number multiplied by the `Delay` will be used.
```

Comment:
Maybe we should add something like `attempt^2 * delay` to these bits of text to make the meaning a bit clearer in addition to the text?